src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -80,56 +66,32 @@ public static String getSlotTrigger(String slotName) {
   }
 
   static {
+    SPECIAL_PAGE_SLOTS_SELECTORS = new HashMap<>();
+    SPECIAL_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)